### ISSUE TITLE
"checked" checkbox not parsed correctly

### ISSUE THREAD

**Issue by metronomme** [AlexxNB/svelte-preprocess-markdown#11]
Hi. I love your work with this project, thank you so much.
I was testing the markdown capabilities today and realised that the preprocessor is stripping off the "checked" and "disabled" attributes of checkbox items.
I tested it on the marked [playground](https://marked.js.org/demo/?text=-%20%5B%20%5D%20test%0A-%20%5Bx%5D%20test&options=%7B%0A%20%22breaks%22%3A%20true%2C%0A%20%22gfm%22%3A%20true%2C%0A%20%22headerIds%22%3A%20true%2C%0A%20%22sanitize%22%3A%20false%2C%0A%20%22smartLists%22%3A%20true%2C%0A%20%22smartypants%22%3A%20true%2C%0A%20%22xhtml%22%3A%20true%0A%7D&version=master) and it works just as expected.
However, when testing on your playground, the following md code:
```
* [ ] unchecked
* [x] checked
```
Produces two unchecked, non-disabled inputs.
Also, I am working with `xhtml: true` but I noticed that the self-closing slash is not produced.... maybe it is all related?

I tried to look at your source code to see where the problem is, but I couldn't get my head around it (I am not so experienced as a programmer! Sorry!)

Thanks again and I hope you can fix that soon!!

**Comment by AlexxNB** [comment]
Ok, I'll check this. Looks strange - this should be proceeded by Marked as usual.

**Comment by AlexxNB** [comment]
Tested this case. Marked in preprocessor produce this:
```html
<ul>
  <li><input disabled="" type="checkbox"> unchecked</li>
  <li><input checked="" disabled="" type="checkbox"> checked</li>
</ul>
```
It is same as example on the Marked Demo Page.

But Svelte parse `checked=""` attribute as `falsy` and remove it attribute from the element.

I'll try to fix this by using custom checkbox renderer in Marked.

**Comment by AlexxNB** [comment]
Fixed in v2.7.2
